doc "Thrown when a mathematical operation caused a number to overflow from its bounds."
shared class OverflowException()
        extends Exception("Numeric overflow", null) {
}